### PR TITLE
chore(curriculum): add quotes to strings in Dice Game

### PR DESCRIPTION
**Pull request by Supravisor**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Related to #51036

### CODE REVIEW

**Review by Jay-Karia — COMMENTED** [review]
In my opinion, I do not think so to add quotes around the strings, as they are already displayed with ``

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/review-algorithmic-thinking-by-building-a-dice-game/657a0ea50da0c8d9d6d7950a.md [hunk 1]
```diff
@@ -9,9 +9,9 @@ dashedName: step-1
...
+The HTML and CSS have been provided for you. Feel free to explore them. When you are ready, you will need to set up your HTML variables. Get all of your `".die"` elements and assign them to a `listOfAllDice` variable. Get your score inputs (the `input` elements in your `"#score-options"`) and score spans, and assign them to `scoreInputs` and `scoreSpans`. Assign the `"#current-round"` element to `currentRound` and the `"#current-round-rolls"` element to `currentRoundRolls`, then do the same for your `"#total-score"` and `"#score-history"` elements. Assign your `"#roll-dice-btn"`, `"#keep-score-btn"`, `#rules-btn`, and `.rules-container` to variables with properly formatted names.
```
Comment (moT01) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/review-algorithmic-thinking-by-building-a-dice-game/657a0ea50da0c8d9d6d7950a.md:12]:
```suggestion
The HTML and CSS have been provided for you. Feel free to explore them. When you are ready, you will need to set up your HTML variables. Get all of your `".die"` elements and assign them to a `listOfAllDice` variable. Get your score inputs (the `input` elements in your `"#score-options"`) and score spans, and assign them to `scoreInputs` and `scoreSpans`. Assign the `"#current-round"` element to `currentRound` and the `"#current-round-rolls"` element to `currentRoundRolls`, then do the same for your `"#total-score"` and `"#score-history"` elements. Assign your `"#roll-dice-btn"`, `"#keep-score-btn"`, `"#rules-btn"`, and `".rules-container"` to variables with properly formatted names.
```

These few should have quotes too?

**Review by moT01 — COMMENTED** [review]
Left one suggestion there.

Also noticed that on step 6, there’s a code block: `, score = ${score}` - should that one have quotes? I'm starting to wonder if the quotes are going to help or just make it more confusing.  

Similar on step 9.

**Review by moT01 — APPROVED** [review]

**Review by huyenltnguyen — COMMENTED** [review]
@Supravisor Thank you for the PR.

I gave this another look and I think we should revisit the changes.

The main goal of #51036 is to avoid confusion when it comes to string values. An example is we ask campers to pass the string `hello` to a function. Without quotation marks, campers would think we expect:

```js
myFunc(hello)
```

This means the following changes are redundant:
  - `"Show rules"` button, `"Roll the dice" button`, `"Four of a Kind"` option`, etc.
  - `".die"` elements`, "#current-round"` element, `"#score-options"` etc.

They are just element "names", and campers would understand what the instructions refer to even without the quotation marks.

---
The same suggestion was mentioned in https://github.com/freeCodeCamp/freeCodeCamp/pull/55676#issuecomment-2268326985.

**Review by huyenltnguyen — APPROVED** [review]

**Pull request merged**